index.html

```diff
@@ -8,7 +8,7 @@
   <title>Hang In There</title>
 </head>
 <body>
-  <section class="main-poster">
+  <section class="main-poster hidden">
```

Comment:
The main poster has been set to `hidden` for debugging! This class can be removed once debugging issues are resolved